### PR TITLE
follow `en-US` to redirect to correct URI

### LINKED ISSUE #5018: Issue with "合作异步JavaScript: 超时和间隔": (links fail)
MDN URL: https://developer.mozilla.org/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous_ae5a561b0ec11fc53c167201aa8af5df

#### What information was incorrect, unhelpful, or incomplete?

Many active learning links fail

#### Specific section or headline?
Active learning: reaction game

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `zh-cn/conflicting/learn/javascript/asynchronous_ae5a561b0ec11fc53c167201aa8af5df`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous_ae5a561b0ec11fc53c167201aa8af5df
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/conflicting/learn/javascript/asynchronous_ae5a561b0ec11fc53c167201aa8af5df/index.html
* Last commit: https://github.com/mdn/translated-content/commit/8824afb494e5398bc0efcf5f7eb78782096fc90c
* Document last modified: 2022-03-18T13:53:46.000Z

</details>很多主动学习的链接失效

#### Did you test this? If so, how?

<details>
<summary>MDage report details</summary>

* Folder: `zh-cn/conflicting/learn/javascript/asynchronous_ae5a561b0ec11fc53c167201aa8af5df`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous_ae5a561b0ec11fc53c167201aa8af5df
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/conflicting/learn/javascript/asynchronous_ae5a561b0ec11fc53c167201aa8af5df/index.html
* Last commit: https://github.com/mdn/translated-content/commit/8824afb494e5398bc0efcf5f7eb78782096fc90c
* Document last modified: 2022-03-18T13:53:46.000Z

</details>

### PR DESCRIPTION
**Pull request by yin1999**
In `en-US`, the URI `Learn/JavaScript/Asynchronous/Timeouts_and_intervals` has been redirected to `Learn/JavaScript/Asynchronous`.

Follow this changes in `zh-CN`.

Fixes #5018

### CODE REVIEW

File: files/zh-cn/_redirects.txt [hunk 1]
```diff
@@ -1098,7 +1098,7 @@
 /zh-CN/docs/Learn/JavaScript/Asynchronous/Async_await	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous/Promises
 /zh-CN/docs/Learn/JavaScript/Asynchronous/Choosing_the_right_approach	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous
 /zh-CN/docs/Learn/JavaScript/Asynchronous/Concepts	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous/Introducing
-/zh-CN/docs/Learn/JavaScript/Asynchronous/Timeouts_and_intervals	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous_ae5a561b0ec11fc53c167201aa8af5df
```
Comment (yin1999) [inline files/zh-cn/_redirects.txt:1101]: For those four redirects:

| origin | redirect |
| ------ | --------- |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Async_await | /zh-CN/docs/Learn/JavaScript/Asynchronous/Promises |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Choosing_the_right_approach | /zh-CN/docs/Learn/JavaScript/Asynchronous |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Concepts | /zh-CN/docs/Learn/JavaScript/Asynchronous/Introducing |
| /zh-CN/docs/Learn/JavaScript/Asynchronous/Timeouts_and_intervals | /zh-CN/docs/Learn/JavaScript/Asynchronous |

File: files/zh-cn/_redirects.txt [hunk 2]
```diff
@@ -2401,7 +2401,7 @@
 /zh-CN/docs/learn/JavaScript/异步/Promises语法	/zh-CN/docs/Learn/JavaScript/Asynchronous/Promises
 /zh-CN/docs/learn/JavaScript/异步/概念	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous/Introducing
 /zh-CN/docs/learn/JavaScript/异步/简介	/zh-CN/docs/Learn/JavaScript/Asynchronous/Introducing
-/zh-CN/docs/learn/JavaScript/异步/超时和间隔	/zh-CN/docs/conflicting/Learn/JavaScript/Asynchronous_ae5a561b0ec11fc53c167201aa8af5df
```
Comment (yin1999) [inline files/zh-cn/_redirects.txt:2404]: redirect here:

| origin | redirect |
| ------ | --------- |
| /zh-CN/docs/learn/JavaScript/异步/Async_await | /zh-CN/docs/Learn/JavaScript/Asynchronous/Promises |
| /zh-CN/docs/learn/JavaScript/异步/Choosing_the_right_approach | /zh-CN/docs/Learn/JavaScript/Asynchronous |
| /zh-CN/docs/learn/JavaScript/异步/概念 | /zh-CN/docs/Learn/JavaScript/Asynchronous/Introducing |
| /zh-CN/docs/learn/JavaScript/异步/超时和间隔 | /zh-CN/docs/Learn/JavaScript/Asynchronous |

**Review by yin1999 — COMMENTED** [review]
changes

**Comment by irvin** [comment]
please check if the redirect works correctly after it's got deploy.

**Comment by yin1999** [comment]
> please check if the redirect works correctly after it's got deploy.

Tested, it's ok.

**Pull request merged**